samples/snippets/fsharp/azure/blob-storage.fsx

```diff
@@ -10,11 +10,6 @@ open System.Text
 //
 
 let storageConnString = "..." // fill this in from your storage account
-(*
-// To fetch the connection string from a configuration file.
-let storageConnString = 
-    CloudConfigurationManager.GetSetting("StorageConnectionString")
-*)
 
 //
 // Create some local dummy data.
```

Comment:
For the `ListBlobsSegmentedInFlatListing` method on L82-99,  to Isaac's point, the method being used,  `GetBlobsByHierarchy` is not asynchronous.   We should remove the `async` in there.